todo_app/app.py

```diff
@@ -1,75 +1,69 @@
 from flask import Flask, render_template, request, redirect, url_for
-from todo_app.data.session_items import get_items, add_item, get_item, save_item, delete_item
+from todo_app.data.trello_items import get_items, add_item, item_in_progress, item_completed, reset_item_status
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
     """
-    Returns the list of saved todo items and sorts by status in descending order. Sorts the item list by status ("Not Started", then "Completed")
-
+    Returns the list of saved todo items.
     """
     items = get_items()
-    sorted_items = sorted(items, key=lambda item: item["status"], reverse=True)
-    return render_template("index.html", items=sorted_items)
+    return render_template("index.html", items=items)
 
 @app.route('/item', methods = ['POST'])
-def add_task():
+def add_new_item():
     """
-    Returns the list of saved todo items. Redirects the user back to the index page
+    Returns the list of saved todo items from Trello web app. Redirects the user back to the index page
 
     """
     if request.method == "POST":
-        new_item = request.form.get("title")
-        add_item(new_item)
+        title = request.form.get("title")
+        desc = request.form.get("desc text")
+        due = request.form.get("date")
+        add_item(title, desc, due)
         return redirect(url_for('index'))
     else:
         return render_template("index.html")
```

Comment:
This case should never arise, since the route itself limits to POST requests (try visiting <http://localhost:5000/item> directly in your browser - you should see a Method Not Allowed error since you're trying to GET this endpoint)